src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx

```diff
@@ -60,27 +60,34 @@ describe('AccordionItemPanel', () => {
       );
 
       expect(
-        wrapper
-          .find(AccordionItemPanel)
-          .find('div')
-          .hasClass('accordionPanelDefault')
-      ).toBe(true);
-      expect(wrapper.find(AccordionItemPanel).find('div').hasClass('foo')).toBe(
-        true
-      );
+        container
+          .querySelector('.accordionDefault')
+          ?.querySelector('div')
+          ?.querySelector('.accordionPanelDefault')
+      ).toBeTruthy();
+
+      expect(
+        container
+          .querySelector('.accordionDefault')
+          ?.querySelector('div')
+          ?.querySelector('.accordionPanelDefault')
+          ?.classList.contains('foo')
+      ).toBeTruthy();
     });
```

Comment:
Same as the comment I mentioned above about `querySelector`. If we can do it with just one, then why not?

Reply:
That's done